### ISSUE TITLE
[Feature]: BiometricManager from Bot API 7.2

### ISSUE THREAD

**Issue by kewldan** [Telegram-Mini-Apps/tma.js#286]
### Describe the solution you'd like

Add BiometricManager implementation

### Additional context

https://core.telegram.org/bots/webapps#biometricmanager

**Comment by heyqbnk** [comment]
Will be implemented in @tma.js/sdk3.0 in the next 1-2 weeks

**Comment by heyqbnk** [comment]
Got confused a bit with versioning. Just released a new major version, its 2.0, but have no docs at the moment. Nevertheless, you can use the JSDoc.

Example:

```ts
import { initBiometryManager } from '@tma.js/sdk';

const bm = await initBiometryManager();

if (!bm.available) {
  // Biometry is not available on the device.
  return;
}

// Do something else with the BiometryManager.
```

**Comment by heyqbnk** [comment]
Docs: https://docs.telegram-mini-apps.com/packages/tma-js-sdk/components/biometry-manager
